src/bindings/python/src/pyopenvino/graph/passes/pattern_ops.cpp

```diff
@@ -553,25 +550,61 @@ static void reg_pattern_optional(py::module m) {
     )");
 
     optional_type.def(
-        py::init([](const std::vector<std::string>& type_names, const std::shared_ptr<ov::Node>& input) {
-            return std::make_shared<ov::pass::pattern::op::Optional>(get_types(type_names), input, nullptr);
+        py::init([](const std::vector<std::string>& type_names, const ov::OutputVector& inputs) {
+            return std::make_shared<ov::pass::pattern::op::Optional>(get_types(type_names), inputs, nullptr);
         }),
         py::arg("type_names"),
-        py::arg("input"),
+        py::arg("inputs"),
         R"(
         Create Optional with the given node type and input node.
 
         :param type_names: node type. For example: ["opset8.Abs", "opset8.Relu"]
         :type type_names: List[str]
 
+        :param inputs: input node's output vector.
+        :type inputs: List[openvino.runtime.Output]
+    )");
+
+    optional_type.def(py::init([](const std::vector<std::string>& type_names, const std::shared_ptr<ov::Node>& input) {
+                          return std::make_shared<ov::pass::pattern::op::Optional>(get_types(type_names),
+                                                                                   ov::as_output_vector({input}),
+                                                                                   nullptr);
+                      }),
+                      py::arg("type_names"),
+                      py::arg("input"),
+                      R"(
+        Create Optional with the given node type and input node.
+
+        :param type_names: node type. For example: ["opset8.Abs", "opset8.Relu"]
+        :type type_names: List[str]
+
         :param input: input node
         :type input: openvino.runtime.Node
     )");
```

Comment:
I think, we're missing a constructor with `(const std::vector<std::string>& type_names, const std::shared_ptr<ov::Node>& input, const Predicate& predicate)`